packages/slate-theme/src/templates/cart.liquid

```diff
@@ -24,10 +24,15 @@
         {% endcomment %}
 
           <tr class="responsive-table-row">
+
             <td data-label="{{ 'customer.order.product' | t }}">
-              <a href="{{ item.url | within: collections.all }}">
-                <img src="{{ item | img_url: '240x240' }}" alt="{{ item.title | escape }}">
-              </a>
+
+              {% if item.image != blank %}
+                <a href="{{ item.url | within: collections.all }}">
+                  <img src="{{ item | img_url: '240x240' }}" alt="{{ item.title | escape }}">
```

Comment:
There's minor inconsistencies in the way we use image tags in Slate. Above, we're using:

```
{{ product.featured_image.src | img_url: '480x480' | img_tag: product.title }}
```

While other instances we're using the HTML `<img>` tag - most likely due to the fact that with the former we're not able to pass a data attribute as a parameter. We could update this to keep it more consistent when not using a data attribute.